.github/PULL_REQUEST_TEMPLATE

```diff
@@ -39,14 +39,10 @@
 
 **Checklist:** :ballot_box_with_check:
 
-<!--
-  Prefix your pull request title with [WIP] if any of these are not complete yet
-
-  Tick non-applicable items as well
--->
+<!-- Leave non-applicable items unchecked -->
 
 - [ ] Updated the documentation for feature additions and enhancements
 - [ ] Added tests for bug fixes or features
 - [ ] Linked all related issues
-- [ ] No blatantly unrelated changes
+- [ ] No unrelated changes <!-- Its tempting, but increases the reviewer's work, and really pollutes the commit history =( -->
 - [ ] Pinged someone for a review!
```

Comment:
Wonder if this item should be removed as well, since its mostly automatic anyway. 🤔
(can replace with a reasonable timeframe (1-2 weeks) then advisory to ping maybe)

Also less stuff to tick.

Reply:
Yes I agree. Also, maybe we can add that for urgent PRs, can just ping someone directly. What do you think?

Reply:
sounds good 👍

Reply:
Hi, could someone explain 'pinged' in this context 😅 ? Is it to use Github's request for review function or just tagging someone? At least for new developers, there is no option to request for PR review. Thanks!

Reply:
@tlylt Yes to both -- but this option will be removed in the future PR template it seems. 

But you can ping us at @MarkBind/active-devs for a PR review :-)